addon/session-stores/adaptive.js

```diff
@@ -80,6 +80,18 @@ export default Base.extend({
   cookieName: proxyToInternalStore(),
 
   /**
+    The path to use for the cookie, e.g., "/", "/something". If not
+    explicitly set, the cookie domain defaults to "/".
+
+    @property cookiePath
+    @type String
+    @default '/'
+    @public
+  */
+  _cookiePath: '/',
```

Comment:
It might actually make sense to default this to [Ember CLI's `rootURL`](https://github.com/ember-cli/ember-cli/issues/4905).

/cc @Turbo87